src/User/resources/views/settings/_menu.php

```diff
@@ -41,6 +41,10 @@
                 'items' => [
                     ['label' => Yii::t('usuario', 'Profile'), 'url' => ['/user/settings/profile']],
                     ['label' => Yii::t('usuario', 'Account'), 'url' => ['/user/settings/account']],
+                    [
+                        'label' => Yii::t('usuario', 'Session history'),
+                        'url' => ['/user/settings/session-history']
```

Comment:
This should be visible only if `enableSessionHistory` is true

Reply:
Fix it.